modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/writer/ColumnValuesExtractor.java

```diff
@@ -323,7 +324,7 @@ public Object apply(Object object)
 		}
 
 		ObjectValuePair<Field, Method> propertiesObjectValuePair =
-			fieldNameObjectValuePairs.get("properties");
+			fieldNameObjectValuePairs.get("lazyProperties");
```

Comment:
I think here we need to consider both cases, if there are no lazy properties, default to properties.